src/js/tabs/firmware_flasher.js

```diff
@@ -330,6 +328,8 @@ firmware_flasher.initialize = function (callback) {
             }
 
             self.releaseLoader.loadTarget(target, release, onTargetDetail);
+
+            self.releaseLoader.loadOptions(release, buildOptions);
```

Comment:
Found the problem. We need to update line 611 as this was introduced in #3332

Reply:
Done.

Reply:
In reality I could probably simplify this a little and simple ask for the options for a given build key, setting the defaults as necessary. Save a round trip. I will do that in a subsequent PR.